src/prometheus/definition.go

```diff
@@ -137,6 +137,18 @@ func FromLabelsValueEntityIDGeneratorForPendingPods() definition.EntityIDGenerat
 	}
 }
 
+func FromLabelGetNamespace() definition.NamespaceGetterFunc {
+	return func(metrics definition.RawMetrics) string {
+		for _, metric := range metrics {
+			m, ok := metric.(Metric)
+			if ok {
+				return m.Labels["namespace"]
+			}
+		}
+		return ""
+	}
+}
```

Comment:
As it does no receive any arguments, it could be defined like this:

```go
func FromLabelGetNamespace(metrics definition.RawMetrics) string {
// ...
}
```
and then, reference the static function `prometheus.FromLabelGetNamespace` (without `()`) in the spec file.

WDYT?

Reply:
yes, fair enough.

Reply:
done